src/ddd.java

```diff
@@ -3,6 +3,6 @@
 
 	public static void main(String args[]){
 		
-		System.out.print("Hello world");

```

Comment:
Why do we need this change?

Reply:
This is needed because we want to change a value